modules/rhizo_base/manifests/apt.pp

```diff
@@ -12,7 +12,22 @@
 #
 
 class rhizo_base::apt {
+  # TODO Switch to "contains" once puppet3 support is no longer required.
```

Comment:
puppet 3 support is not required here.

Reply:
Please go ahead and do it the right way for puppet 4+

Reply:
Will do!